### ISSUE TITLE
circle.yml help and syntax

### ISSUE THREAD

**Issue by levlaz** [jvandyke/vsc-circleci#1]
Hey! Thanks so much for making this plugin. I just switched over to VS Code and its awesome. 

I notice this listed on the upcoming features, I think this would be a huge help. @projectfrank started working on a general purpose circle.yml validator, I wonder if we could collaborate to make this feature.

**Comment by jvandyke** [comment]
Yeah, I think that'd be great. Is there an existing library for validation? Or at least a grammar/schema of all available options?

I'd like to see this as a generic NPM module, then that can be incorporated into this plugin once we know what's available.
